cp-ksql-server/Dockerfile

```diff
@@ -0,0 +1,33 @@
+ARG DOCKER_REGISTRY
+
+FROM ${DOCKER_REGISTRY}confluentinc/cp-base
```

Comment:
Just for confirmation: We don't need a `/` between `${DOCKER_REGISTRY}` and `confluentinc/...`?

Reply:
To make sure the correct per-branch base image is getting pulled, this should start off as:

ARG DOCKER_UPSTREAM_REGISTRY
ARG DOCKER_UPSTREAM_TAG

FROM ${DOCKER_UPSTREAM_REGISTRY}confluentinc/cp-base${DOCKER_UPSTREAM_TAG}

Reply:
updated @aayars 

@miguno we don't need the '/', it is added by the build system when the arg is passed in.